lib/playground.dart

```diff
@@ -80,11 +92,29 @@ class Playground {
       if (!isMobile()) _context.focus();
     });
 
-    // TODO: Currently the lights are all shared; we should have one for each
-    // type.
-    dartBusyLight = new DBusyLight(querySelector('#dartbusy'));
-    cssBusyLight = new DBusyLight(querySelector('#dartbusy'));
-    htmlBusyLight = new DBusyLight(querySelector('#dartbusy'));
+    busyLight = new DBusyLight(querySelector('#dartbusy'));
+
+    // Update the title on changes.
+    titleEditable = new DContentEditable(
+        querySelector('header .header-gist-name'));
+    bind(titleEditable.onChanged, editableGist.property('description'));
+    bind(editableGist.property('description'), titleEditable.textProperty);
+
+    // Update the ID on changes.
+    AnchorElement idAnchor = querySelector('header .header-gist-id a');
+    bind(editableGist.property('id'), (val) => idAnchor.text = val);
+    bind(editableGist.property('html_url'), (val) {
+      idAnchor.href = val == null ? '' : val;
+    });
+
+    Throttler throttle = new Throttler(const Duration(milliseconds: 100));
```

Comment:
Comment out the object to correspond to the usage?

Could you add a short comment at the top of the disabled block?